components/Internship/NextSteps.js

```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+function NextSteps(props) {
+    const { image, name, info } = props;
+    const [isFlipped,setFlipped] = useState(true);
+    return (
+        <div className="committee-card" onClick={() => setFlipped(!isFlipped)}>
+            <div className={isFlipped ? 'next-steps-card-inner' : 'next-steps-card-inner is-flipped'}>
+                <div className='next-steps-card-face'>
+                    <Image
+                        src={image}
+                        alt={`${name}'s card motif`}
```

Comment:
motif is not a great alt text tag, we might want to workshop this!

Reply:
hmm i wasn't really sure what to name it but yeah motif probably isn't a great alt. Not sure how to come up with a generalized alt for this other than using 'motif' or 'image' though. Maybe I can write descriptions for each card and add an `altTextCard` property to the js data file to be used as the alt tag?

Reply:
So the idea is that if the image doesn't render (or the user cannot see the image), then the system should be usable. With that in mind, "motif" is not actually the relevant description: it should be something along the lines of "clicking this will let you see more information".